apps/07jquery-tic-tac-toe/script.js

```diff
@@ -1,5 +1,40 @@
 'use strict';
 
+
 $(document).on('ready', function() {
-    // Put app logic in here
+  var playerTurn = 'X';
+  $('[data-cell]').click(function() {
+    $(this).text(playerTurn);
+    checkForWin();
```

Comment:
OO this ^ is actually a good warning. Do you understand what it's complaining about?